src/conduits/RwaOutputConduit3.sol

```diff
@@ -0,0 +1,261 @@
+     */
+    constructor(address _psm, address _quitTo) public {
+        DSTokenAbstract _gem = DSTokenAbstract(GemJoinAbstract(PsmAbstract(_psm).gemJoin()).gem());
+        psm = PsmAbstract(_psm);
+        dai = DSTokenAbstract(PsmAbstract(_psm).dai());
+        gem = _gem;
+        quitTo = _quitTo;
+        toGemConversionFactor = 10**(dai.decimals() - _gem.decimals());
+
+        // Give unlimited approve to PSM gemjoin
+        dai.approve(_psm, type(uint256).max);
+
+        wards[msg.sender] = 1;
+        emit Rely(msg.sender);
+    }
+
+    modifier auth() {
+        require(wards[msg.sender] == 1, "RwaOutputConduit3/not-authorized");
+        _;
+    }
+
+    modifier isMate() {
+        require(may[msg.sender] == 1, "RwaOutputConduit3/not-mate");
+        _;
+    }
+
+    /*//////////////////////////////////
+               Authorization
+    //////////////////////////////////*/
+
+    /**
+     * @notice Grants `usr` admin access to this contract.
+     * @param usr The user address.
+     */
+    function rely(address usr) external auth {
+        wards[usr] = 1;
+        emit Rely(usr);
+    }
+
+    /**
+     * @notice Revokes `usr` admin access from this contract.
+     * @param usr The user address.
+     */
+    function deny(address usr) external auth {
+        wards[usr] = 0;
+        emit Deny(usr);
+    }
+
+    /**
+     * @notice Grants `usr` push access to this contract.
+     * @param usr The user address.
+     */
+    function mate(address usr) external auth {
+        may[usr] = 1;
+        emit Mate(usr);
+    }
+
+    /**
+     * @notice Revokes `usr` push access from this contract.
+     * @param usr The user address.
+     */
+    function hate(address usr) external auth {
+        may[usr] = 0;
+        emit Hate(usr);
+    }
+
+    /**
+     * @notice Grants `usr` operator access to this contract.
+     * @param usr The user address.
+     */
+    function hope(address usr) external auth {
+        can[usr] = 1;
+        emit Hope(usr);
+    }
+
+    /**
+     * @notice Revokes `usr` operator access from this contract.
+     * @param usr The user address.
+     */
+    function nope(address usr) external auth {
+        can[usr] = 0;
+        emit Nope(usr);
+    }
+
+    /*//////////////////////////////////
+               Administration
+    //////////////////////////////////*/
+
+    /**
+     * @notice Updates a contract parameter.
+     * @param what The changed parameter name. `"quitTo", "to"`
+     * @param data The new value of the parameter.
+     */
+    function file(bytes32 what, address data) external auth {
+        if (what == "quitTo") {
+            require(data != address(0), "RwaOutputConduit3/invalid-quit-to-address");
+            quitTo = data;
+        } else if (what == "to") {
+            require(data != address(0), "RwaOutputConduit3/invalid-to-address");
+            to = data;
+        } else {
+            revert("RwaOutputConduit3/unrecognised-param");
+        }
```

Comment:
I'm not sure if we should follow this pattern here.
All other conduits have a `pick` function, we shouldn't remove it for compatibility sake.

Reply:
I don't have any preference here. I raise this question in dev channel earlier today

Reply:
Done

Reply:
Sorry about that... after today's discussion, maybe `file` is the best approach.

Reply:
No worries, already revert it